From 2fc66561df46e3a757845ebd9a2a0f11b3b46c6b Mon Sep 17 00:00:00 2001
From: Andrii Osipov <aosipov@anaconda.com>
Date: Thu, 22 May 2025 00:56:24 -0700
Subject: [PATCH] disable sphinx_toolbox in conftest

---
 tests/conftest.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/conftest.py b/tests/conftest.py
index 2c157be..11fa612 100644
--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -19,7 +19,7 @@ else:
 	# 3rd party
 	from sphinx.testing.path import path  # type: ignore[assignment]
 
-pytest_plugins = ("coincidence", "sphinx.testing.fixtures", "sphinx_toolbox.testing")
+pytest_plugins = ("coincidence", "sphinx.testing.fixtures") #, "sphinx_toolbox.testing")
 
 
 @pytest.fixture(scope="session")
-- 
2.39.3 (Apple Git-146)

